app/assets/javascripts/bulk_updates.coffee

```diff
@@ -25,35 +41,37 @@ $(document).ready ->
 
         # matchingKeywords = $(keywordList).find('li').filter ->
         #   this.childNodes[0].nodeValue.trim() == value
-        keywordLengthMinimum = 2
+        keywordLengthMinimum = if type == 'science' then 2 else 1
```

Comment:
We had discussed setting this as an option on the picker, I think that would be a meaningful improvement.